src/Api/Tools/Models/Response/MemberAccessReportModel.cs

```diff
@@ -0,0 +1,150 @@
+﻿using Bit.Core.AdminConsole.Entities;
+using Bit.Core.Entities;
+using Bit.Core.Models.Data;
+using Bit.Core.Models.Data.Organizations;
+using Bit.Core.Models.Data.Organizations.OrganizationUsers;
+using Bit.Core.Vault.Models.Data;
+
+namespace Bit.Api.Tools.Models.Response;
+
+/// <summary>
+/// Member access details. The individual item for the detailed member access
+/// report. A collection can be assigned directly to a user without a group or
+/// the user can be assigned to a collection through a group. Group level permissions
+/// can override collection level permissions.  
+/// </summary>
+public class MemberAccessReportAccessDetails
+{
+    public Guid CollectionId { get; set; }
+    public Guid? GroupId { get; set; }
+    public string GroupName { get; set; }
+    public string CollectionName { get; set; }
+    public int ItemCount { get; set; }
+    public bool ReadOnly { get; set; }
+    public bool HidePasswords { get; set; }
+    public bool Manage { get; set; }
+
+    // internal to not expose 
+    internal Guid? UserGuid { get; set; }
```

Comment:
My [previous comment](https://github.com/bitwarden/server/pull/4599#discussion_r1714527946) has been partially resolved but I still have the same question for this remaining property.